### PR TITLE
Filter index tables

### PR DESCRIPTION
**Pull request by bfpepper**
Can now filter both index tables.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,15 +1,89 @@
+var $ = require('jquery')
 var Food = require('./foods');
 var Exercise = require('./exercises');
 
 var someFood = new Food();
 var someExercise = new Exercise();
 
+setDiaryExercisesTable = function () {
```
Comment (AliSchlereth) [inline lib/index.js:8]: I like the exposure to doing this with vanilla JS.
I think we should refactor this into a Filter class at some point. 
@bfpepper

File: test/user-creates-an-exercise-entry-test.js [hunk 2]
```diff
@@ -13,13 +13,13 @@ test.describe("user creates an exercise entry from exercises.html", function(){
   })
 
   test.afterEach(function(){
-    driver.get('http://localhost:8080/webpack-dev-server/exercises.html')
+    driver.get('http://localhost:8080/exercises.html')
```
Comment (AliSchlereth) [inline test/user-creates-an-exercise-entry-test.js:16]: So many hours!!!

**Comment by AliSchlereth** [comment]
It says there is a merge conflict on this. Maybe now that the testing branch is pushed? I'll be offline until after wrap-up. I'll try to check in in about 20min if possible.  Leading a spike this afternoon so it may be tough.  @bfpepper

**Comment by bfpepper** [comment]
@AliSchlereth Fixed the conflict, re-pushed, should be all good now.

**Comment by AliSchlereth** [comment]
Thanks @bfpepper !

**Pull request merged**